Packages/vcs/Lib/VTKPlots.py

```diff
@@ -214,13 +229,22 @@ def configureEvent(self,obj,ev):
     if self.canvas.animate.created():
       self.canvas.animate.draw_frame()
 
+    if manager:
+      # Set the UI renderer's layer on top of what's there right now
+      self.setLayer(manager.renderer, 3)
```

Comment:
Yup, that was on the list of things to do that I totally forgot about. I'll do that now.